docs.json

```diff
@@ -114,7 +114,15 @@
                 "pages": [
                   "services/registry/overview",
                   "services/registry/foundations",
-                  "services/registry/developer-guide",
+                  {
+                    "group": "Developer Guide",
+                    "pages": [
+                      "services/registry/developer-guide",
+                      "services/registry/cli-setup",
+                      "services/registry/data-collection-developer-guide",
+                      "services/registry/edge-function-developer-guide"
+                    ]
+                  },
```

Comment:
not sure it's useful to put these pages into a new group. I suggest to add them to the parent group directly

Reply:
done!